packages/manager/modules/exchange/src/index.js

```diff
@@ -28,6 +29,7 @@ angular
   )
   .service('APIExchange', APIExchange)
   .service('Exchange', Exchange)
-  .service('ExchangePassword', ExchangePassword);
+  .service('ExchangePassword', ExchangePassword)
+  .service('navigation', navigation);
```

Comment:
the service name is too generic to be exported like that no ?

Reply:
We could rename but actually it's already exported as this. 
It was available before adding lazy loading to the exchange root state, and now this `navigation` service is only available after reaching the first exchange route.